src/ios/CleverTapPlugin.m

```diff
@@ -1386,8 +1382,8 @@ - (void)reset {
 }
 
 - (void)setLibrary: (CDVInvokedUrlCommand *)command {
```

Comment:
Will this function be called when the Plugin is initialised? @AishwaryaNanna

Reply:
@AishwaryaNanna call the method on pluginInitialize or the entry point.